src/pages/account/sign-up-or-sign-in.tsx

```diff
@@ -84,9 +84,12 @@ export const getServerSideProps: GetServerSideProps = withContainer(
 
 const validationRules = ({ signUpOrSignIn }) => {
   return new FormManager({
-    signUpOrSignIn: new FieldManager(signUpOrSignIn, [
-      Validators.required("Please select an option"),
-    ]),
+    signUpOrSignIn: new FieldManager(
+      signUpOrSignIn,
+      [Validators.required("Please select an option")],
+      undefined,
```

Comment:
Default to empty array rather than undefined as this is what the argument paramater defaults to